_includes/js/schemas.md

```diff
@@ -0,0 +1,156 @@
+# Schema
+
+Schema is the structure representing classes in your app. You can use the schema
+of an app to verify operations in a unit test, generate test data, generate test
+classes and then clean up after tests. The schema API can also be used to create
+custom views of your data. We use the schema API to display columns names and
+types in the databrowser.
+
+This API allows you to access the schemas of your app.
+
+* `MasterKey` is required.
+* Starting with Parse-Server 2.7.1
+
+Schema will return an object similar to the following:
+
+<pre><code class="javascript">
```

Comment:
Wait a minute...this is a duplicate file of **schema.md** right? Is there a particular reason why this one (or that one) is being added?

Reply:
Oops